frontend/src/pages/list/BookList.tsx

```diff
@@ -105,9 +106,14 @@ const COLUMNS: TableColumn[] = [
 class BookList extends React.Component<{}, BookListState> {
   constructor(props = {}) {
     super(props);
-    this.state = { value: "" };
+    this.state = { value: "", addIsbns: "" };
   }
 
+  onSubmit = (event: FormEvent<HTMLFormElement>): void => {
```

Comment:
I'm confused about why this is in the BookList class, maybe this is old? Looks like BookAdd code

Reply:
Same thing below for GenreList and VendorList, but maybe I'm not understanding